target/bin/setquota

```diff
@@ -15,38 +15,38 @@ USER="${1}"
 shift
 QUOTA="${*}"
 
-function usage  { echo "Usage: setquota <user@domain> [<quota>]" ; }
+function usage { echo 'Usage: setquota <user@domain> [<quota>]' ; }
 
-[[ -z ${USER} ]] && { usage ; _exit_with_error "no username specified" ; }
-[[ ${USER} =~ .*\@.* ]] || { usage ; _exit_with_error "username must include the domain" ; }
+[[ -z ${USER} ]] && { usage ; _exit_with_error "No username specified" ; }
+[[ ${USER} =~ .*\@.* ]] || { usage ; _exit_with_error "Username must include the domain" ; }
 
 if ! grep -qE "^${USER}\|" "${USER_DATABASE}"
 then
-  usage; _exit_with_error "user ${USER} does not exist"
+  usage; _exit_with_error "User '${USER}' does not exist"
 fi
 
 # check quota
 if [[ -n ${QUOTA} ]] && ! echo "${QUOTA}" | grep -qE "^([0-9]+(B|k|M|G|T)|0)\$"
 then
   usage
-  _exit_with_error "invalid quota format. e.g. 302M (B (byte), k (kilobyte), M (megabyte), G (gigabyte) or T (terabyte))"
+  _exit_with_error 'Invalid quota format. e.g. 302M (B (byte), k (kilobyte), M (megabyte), G (gigabyte) or T (terabyte))'
 fi
 
 _create_lock # Protect config file with lock to avoid race conditions
 
 touch "${DATABASE}"
 if [[ -z ${QUOTA} ]]
 then
-  read -r -s "Enter quota (e.g. 10M): " QUOTA
+  read -r -s 'Enter quota (e.g. 10M): ' QUOTA
   echo
-  [[ -z "${QUOTA}" ]] && _exit_with_error "Quota must not be empty. Use 0 for unlimited quota"
+  [[ -z "${QUOTA}" ]] && _exit_with_error 'Quota must not be empty. Use 0 for unlimited quota'
 fi
 
 # check quota
 if [[ -n ${QUOTA} ]] && ! echo "${QUOTA}" | grep -qE "^([0-9]+(B|k|M|G|T)|0)\$"
 then
   usage
-  _exit_with_error "invalid quota format. e.g. 302M (B (byte), k (kilobyte), M (megabyte), G (gigabyte) or T (terabyte))"
+  _exit_with_error 'Invalid quota format. e.g. 302M (B (byte), k (kilobyte), M (megabyte), G (gigabyte) or T (terabyte))'
 fi
```

Comment:
It looks like this block of code is a 2nd instance. I'm not sure why there are two copies in close proximity of each other.

Probably more maintainable as a function with two calls if having both instances is intentional?

Reply:
I will outsource this into a variable :)

Reply:
Done in 08a05debd67e69c3e5010be547c0cd0ca65a28c7